libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1826,10 +1830,10 @@ std::string YulUtilFunctions::clearStorageRangeFunction(Type const& _type)
 
 	return m_functionCollector.createFunction(functionName, [&]() {
 		return Whiskers(R"(
-			function <functionName>(start, end) {
-				for {} lt(start, end) { start := add(start, <increment>) }
+			function <functionName>(start, slotCount) {
+				for { let i := 0 } lt(i, slotCount) { i := add(i, <increment>) }
```

Comment:
Variable names should include the unit to make things clearer when you read the Yul source:
```suggestion
			function <functionName>(startSlot, slotCount) {
				for { let i := 0 } lt(i, slotCount) { i := add(i, <slotsPerItem>) }
```

And `start` -> `startSlot` in the docstring as well.